msu/hooks/skills/skill.nut

```diff
@@ -33,18 +34,27 @@
 					break;
 
 				case ::Const.Injury.BluntAndPiercingBody:
-					this.m.DamageType.add(::Const.Damage.DamageType.Blunt, 55);
-					this.m.DamageType.add(::Const.Damage.DamageType.Piercing, 45);
+					local blunt = ::Const.Injury.BluntBody.len() + ::Const.Injury.BluntHead.len();
+					local piercing = ::Const.Injury.PiercingBody.len() + ::Const.Injury.PiercingHead.len();
+					local total = (blunt + piercing).tofloat();
+					this.m.DamageType.add(::Const.Damage.DamageType.Blunt, ::MSU.Math.round(100 * blunt / total, 5)); // We round to the nearest multiple of 5 to make it "cleaner" for players
+					this.m.DamageType.add(::Const.Damage.DamageType.Piercing, ::MSU.Math.round(100 * blunt / total, 5));
```

Comment:
Should be `::MSU.Math.round(100 * piercing/ total, 5))`, no?

Reply:
Yes, will fix